packages/type-safe-api/scripts/type-safe-api/generators/python-cdk-infrastructure/templates/functions.handlebars

```diff
@@ -10,6 +10,7 @@
 from aws_cdk.aws_lambda import (
     Function, Runtime, Tracing, Code
 )
+from aws_pdk.type_safe_api import SnapStartFunction
```

Comment:
Shouldn't this be conditional?

Reply:
The generated infra code doesn't error for unused imports so I kept them there for simplicity - otherwise I need to do both "if python" and "if typescript" as there's no "else" - doable but not much gain besides slightly neater generated code :)